content/en/integrations/guide/prometheus-metrics.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Mapping Prometheus Metrics to Datadog Metrics
+kind: guide
+aliases:
+  - /integrations/faq/how-to-collect-metrics-with-sql-stored-procedure/
+further_reading:
+- link: "https://www.datadoghq.com/blog/sql-server-metrics/#create-a-stored-procedure-to-generate-and-collect-metrics"
+  tag: "Blog"
+  text: "Create a stored procedure to generate and collect metrics"
+- link: "/integrations/mysql/"
+  tag: "Documentation"
+  text: "Datadog-MySQL integration"
+---
+
+If you are using Datadog's Prometheus or OpenMetrics checks, you may be interested in how these metrics map to existing Datadog metrics types. 
+
+## Prometheus and OpenMetrics metric types
+
+* `counter`: a cumulative metric that represents a single monotonically increasing counter, whose value can only increase—or be reset to zero.
+* `gauge`: a metric that represents a single numerican value, which can arbitrarily go up and down.
+* `histogram`: samples observations and counts them in configurable buckets; also provides a sum of all observed values.
+* `summary`: similar to histogram; samples observations, provides a sum of all observed values, and calculates configurable quantiles over a sliding time window.
+
+## How they map to Datadog metrics
+
+For more information about Datadog metric types, see the [Metric Types documentation][1].
+
+### Counter
+
+By default, Prometheus/OpenMetrics `counter` is mapped to Datadog's `gauge`.
```

Comment:
I think that we should explain the logic of our transformation here, otherwise users will be confused

Reply:
More specifically, users might wonder why OM 'counter' is not mapped to DD 'count'